test/integration/dss/test_dss_api.py

```diff
@@ -20,15 +21,41 @@
 
 if USING_PYTHON2:
     import backports.tempfile
+    import mock
     import unittest2 as unittest
     TemporaryDirectory = backports.tempfile.TemporaryDirectory
 else:
     import unittest
+    from unittest import mock
     TemporaryDirectory = tempfile.TemporaryDirectory
 
+
 class TestDssApi(unittest.TestCase):
     staging_bucket = "org-humancellatlas-dss-cli-test"
 
+    def test_set_host(self):
+        with TemporaryDirectory() as home:
+            with mock.patch.dict(os.environ, HOME=home):
```

Comment:
Since I wrote these tests, I noticed that `tweak` support `XDG_CONFIG_HOME`. Patching that instead of HOME would be more surgical.

Reply:
Were you planning on addressing this?